### PR TITLE
Update Client Credentials Grant Setup instructions to be more explicit

### PR DESCRIPTION
**Pull request by max-rocket-internet**
For Keycloak beginners like me, this would help a lot 🙂

Also replacing `""` with `` to better show the parts that are Keycloak UI elements.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -17,11 +17,11 @@ sure which to use, the client credentials grant is recommended, as it was design
...
-    1. Set "Access Type" to "confidential".
-    1. Set "Standard Flow Enabled" to "OFF".
-    1. Set "Direct Access Grants Enabled" to "OFF"
-    1. Set "Service Accounts Enabled" to "ON".
-1. Grant required roles for managing Keycloak via the "Service Account Roles" tab in the client you created in step 1.
+    1. Set `Access Type` to `confidential`.
+    1. Set `Standard Flow Enabled` to `OFF`.
+    1. Set `Direct Access Grants Enabled` to `OFF`
+    1. Set `Service Accounts Enabled` to `ON`.
+1. Grant required roles for managing Keycloak via the `Service Account Roles` tab in the client you created in step 1. To do this, select `realm-management` in the `Client Roles` drop down menu and add required roles to the client or just assign `realm-admin` for full access.
```
Comment (mrparkers) [inline docs/index.md:24]: This is actually covered in the section below called "Assigning Roles".  Could you take a look at that and see if it makes sense to tweak that a little bit?  I'm also fine with adding a link here that points to that other section, but I'd prefer not to duplicate its content.

Comment (max-rocket-internet) [reply docs/index.md:24]: Oh you are right! I just didn't read that bit, my fault.

> I'm also fine with adding a link here that points to that other section

Better idea, I'll update the PR.

Comment (max-rocket-internet) [reply docs/index.md]: Done

**Review by mrparkers — APPROVED** [review]
LGTM, thanks!

**Pull request merged**